flat_pack.scad

```diff
@@ -0,0 +1,169 @@
+//openscad flat-pack joint library
+
+module finger_testing() {
```

Comment:
Could this just go in a separate file that demos the way this library work? Maybe shows how all the parameters can be varied and how you can make a mating joint using centered=true vs centered=false